scripts/pylint_extensions.py

```diff
@@ -1252,12 +1253,15 @@ def process_module(self, node):
             # shebang expression at the start of a bash script which
             # loses function when a space is added.
             if re.search(br'^#[^\s].*$', line) and not line.startswith(b'#!'):
+                space_at_beginning = False
                 self.add_message(
                     'no-space-at-beginning', line=line_num + 1)
 
-            # Check if comment contains version info or data type.
-            if not previous_line.startswith(b'#'):
-                data_type_is_present = any(word in line for word in DATA_TYPES)
+            # Check if comment contains version info or data type
+            # at the beginning.
```

Comment:
Please explain the reason why this is important (no capital letter) and also give examples e.g. "v2 version does not have ExplorationStatsModel."

In particular, explain that these are exceptions to the "comment must start with a capital letter" rule.

Reply:
I think it is better to have regex check for version info because it might be anything eg. v1, v2, or v3.

Reply:
You could just have the prefix be "v".

Reply:
Done.